app/containers/Profile/index.js

```diff
@@ -8,29 +8,95 @@ import React from 'react';
 import { connect } from 'react-redux';
 import { createStructuredSelector } from 'reselect';
 import { compose } from 'redux';
+import { Helmet } from 'react-helmet';
+import { translationMessages } from 'i18n';
 
 import injectSaga from 'utils/injectSaga';
 import injectReducer from 'utils/injectReducer';
-import makeSelectProfile from './selectors';
+import PropTypes from 'prop-types';
+
+import { makeSelectLocale } from 'containers/LanguageProvider/selectors';
+import { makeSelectAccount } from 'containers/AccountInitializer/selectors';
+
+import { getProfileInfo, setDefaultProps } from './actions';
+
 import reducer from './reducer';
 import saga from './saga';
+import messages from './messages';
+import * as profileSelectors from './selectors';
+
+import Preloader from './Preloader';
+import Wrapper from './Wrapper';
+import NoSuchUser from './NoSuchUser';
 
 /* eslint-disable react/prefer-stateless-function */
 export class Profile extends React.Component {
+  componentDidMount = () => {
+    /* eslint-disable-next-line */
+    const { id } = this.props.children._owner.stateNode.props.match.params;
```

Comment:
Better approach would be to define prop on the component and pass id down from page component.

Reply:
done